### ISSUE TITLE
HTML5 input type tel

### ISSUE THREAD

**Issue by inetbiz** [AdvancedCustomFields/acf#16]
A new input type must be created in order to collect tel and let the browser format and verify input for submission.  https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/tel

**Comment by elliotcondon** [comment]
Hi @inetbiz 

Thanks for the request. I agree this field type would be a great addition to the plugin, but wonder if this is veering away from "content editing" and more towards "contact forms".

**Comment by inetbiz** [comment]
@elliotcondon I'm buying the dev license. I need the `tel` field to ask for org phone. that input type is the fastest way to ensure correct input. IMHO

**Comment by elliotcondon** [comment]
Hi @inetbiz 

For sure. I agree that the field type will be useful, but am not sure it belongs in the core.
It would be very easy to create this field type yourself as a 3rd party solution. Checkout the tutorial here: https://www.advancedcustomfields.com/resources/creating-a-new-field-type/

**Comment by inetbiz** [comment]
@elliotcondon I strongly disagree. POSH HTML5. If you're going to have an input field of email and url, you also need one for tel.
